controllers/article.js

```diff
@@ -0,0 +1,42 @@
+import 'babel-polyfill';
+import models from '../models';
+import { generateSlug } from '../helpers';
+
+const { Article, User, Profile } = models;
+
+export default {
+  createArticle: async (req, res) => {
+    const {
+      title, body, description, category,
+    } = req.body;
+
+    const userId = req.decoded.id;
+    const user = await User.findOne(
+      {
+        where: { id: userId },
+        include: [{ model: Profile }]
+      });
+
+    const profile = user.Profile;
+    const article = await Article
+      .create({
+        title: title.trim(),
+        body: body.trim(),
+        description: description.trim(),
+        category: category.trim(),
+        slug: generateSlug(title),
+        userId,
+      });
+    return res.status(201).json({
+      title: article.title,
```

Comment:
These are still not trimmed before creation

Reply:
Please by before creation, do you mean I should trim them before declaring `const article`?

Reply:
No. This is fine. This is exactly what I was asking for.

Reply:
Thanks